src/components/template/import/MergeReview.tsx

```diff
@@ -83,70 +123,46 @@ export function MergeReview(props: MergeReviewProps) {
 
   return (
     <>
-      <Card>
-        <CardContent>
-          <Typography variant="body2">Version: {version}</Typography>
-          <Typography variant="body2">Export date: {exportDate}</Typography>
-        </CardContent>
-      </Card>
-      {props.data.questionComparisons.map((comparison) => (
-        <ConflictResolver<QuestionComparison>
-          key={comparison.newQuestion.id}
-          comparison={comparison}
-          onConflictResolved={onConflictResolved}
-          getStatus={(comparison) => comparison.status}
-          getItemId={(comparison) => comparison.newQuestion.id}
-          getItemTitle={(comparison) => comparison.newQuestion.question}
-          getDiffInfo={({ existingQuestion, newQuestion }) => {
-            return [
-              {
-                existingVal: existingQuestion?.naturalKey,
-                newVal: newQuestion?.naturalKey ?? '',
-                heading: 'Natural key',
-                isDifferent:
-                  existingQuestion !== null &&
-                  existingQuestion?.naturalKey !== newQuestion.naturalKey,
-              },
-              {
-                existingVal: existingQuestion?.question,
-                newVal: newQuestion?.question ?? '',
-                heading: 'Question',
-                isDifferent:
-                  existingQuestion !== null &&
-                  existingQuestion.question !== newQuestion.question,
-              },
-              {
-                existingVal: (
-                  <pre>
-                    {JSON.stringify(existingQuestion?.config, undefined, 4) ||
-                      '-'}
-                  </pre>
-                ),
-                newVal: (
-                  <pre>{JSON.stringify(newQuestion?.config, undefined, 4)}</pre>
-                ),
-                heading: 'Config',
-                isDifferent:
-                  existingQuestion !== null &&
-                  deepEqual(existingQuestion?.config, newQuestion.config) ===
-                    false,
-              },
-            ];
-          }}
-        />
-      ))}
+      <Box sx={{ padding: 1 }}>
```

Comment:
This is fine but on the `Box` component you should also have `p` or `padding` as property. Maybe that is a bit smarter.